test/test_loggers.py

```diff
@@ -42,6 +42,48 @@ def test_log_scalar(self, steps):
                 if steps:
                     assert event_acc.Scalars("foo")[i].step == steps[i]
 
+    @pytest.mark.parametrize("steps", [None, [1, 10, 11]])
+    def test_log_video(self, steps):
+        torch.manual_seed(0)
+        with tempfile.TemporaryDirectory() as log_dir:
+            exp_name = "ramala"
+            logger = TensorboardLogger(log_dir=log_dir, exp_name=exp_name)
+
+            # creating a sample video (T, C, H, W), where T - number of frames,
+            # C - number of image channels (e.g. 3 for RGB), H, W - image dimensions.
+            # the first 64 frames are black and the next 64 are white
+            video = torch.cat(
+                (torch.zeros(64, 1, 32, 32), torch.mul(torch.ones(64, 1, 32, 32), 255))
```

Comment:
you can use `torch.full` here for a tensor full of 255

Reply:
yes, thanks!

Reply:
Re the "fps for tensorboard", the problem is that the return format for the video is an encoded_string, which has the same length for any video. I didn't find a nice method to convert it back to tensor or video object.
Re the "want to test that the video looks like what it should": 
- I am testing it in the CSVLogger
- For the WandbLogger and Tensorboard haven't found a nice method to do that.

Reply:
Ok I think we're good to go as it is!